ipe_utils/df_utils.py

```diff
@@ -27,7 +27,7 @@ def df_dosage_clean_up(df: pd.DataFrame) -> pd.DataFrame:
     """
     Dosage values should be  number, and in between 0 and 100.
     """
-    df_dosage_to_numeric = df.loc[df[COL_DOSAGE].apply(pd.to_numeric, errors='coerce').notnull()]
+    df_dosage_to_numeric = df.loc[df[COL_DOSAGE].apply(pd.to_numeric, errors='coerce').notnull()].copy()
```

Comment:
I think the old code without .copy() should also work. But the new code works, too.

Reply:
It will work, but it throws a warning, so this change is to remove the warning.